test/parallel/test-http-set-timeout-server.js

```diff
@@ -169,6 +165,47 @@ test(function idleTimeout(cb) {
       allowHalfOpen: true,
     };
     const c = net.connect(options, () => {
+      c.on('error', (e) => {
+        if (e.message !== 'read ECONNRESET')
+          throw e;
```

Comment:
I'm assuming no `assert.fail` since this is a system error not something we want to assert, right?